bark/runtime/tests/data/centered_city_highway_straight.xodr

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" standalone="yes"?>
```

Comment:
why do we use a different map, would it not be better to fix the other map?

Reply:
May I directly modify city_highway to center it?

Reply:
would be okay for me, if all the tests run through (they should not rely on those global coordinates anyway)